packages/libs/src/sdk/services/EntityManager/types.ts

```diff
@@ -75,7 +77,8 @@ export enum EntityType {
   GRANT = 'Grant',
   DASHBOARD_WALLET_USER = 'DashboardWalletUser',
   TIP = 'Tip',
-  COMMENT = 'Comment'
+  COMMENT = 'Comment',
+  REACTION = 'Reaction'
```

Comment:
Do you think it makes sense to prefix `COMMENT_REACTION`

Reply:
this isn't actually used, ill remove